### DISCUSSION TITLE
[studio] Support multiple results in SQL Editor

### DISCUSSION THREAD

**Discussion by osaxma** [supabase/supabase#9930] [Feature Requests]
# Feature request

## Current limitation

Running multiple queries only shows the result of the first query:

<p align="center"><img width="515" alt="Screen Shot 2022-10-11 at 3 11 19 PM" src="https://user-images.githubusercontent.com/46427323/195086635-b26074d3-f22f-4626-ba65-d3841310a244.png"></p>
<p align="center">Supabase SQL Editor</p>

## Describe the solution you'd like
I would like the SQL editor to show a result for each query such as:
 
<p align="center"><img width="657" alt="Screen Shot 2022-10-11 at 3 13 26 PM" src="https://user-images.githubusercontent.com/46427323/195086689-b019606a-e63d-4470-9a54-71d435871585.png"></p>
<p align="center">SQL Notebook (VS Code extension)</p>

## Describe alternatives you've considered

Using other SQL editors than Supabase's.  

## Additional context

I mainly use the SQL Editor for debugging flow such as: `set role authenticated`, modify `request.jwt` to match user id and role, then execute query to see result. 

Typically I'm interested in the result of the last query but I think supporting all results (within diff tabs or stacked) would be more beneficial.

**Comment by saltcod** [comment]
Hey @osaxma !

Very neat request. I'm going to move this into Discussions and flag it as a feature request. Great idea.
